docs/updates.md

```diff
@@ -0,0 +1,46 @@
+# Updates
+
+## October 2018
+
+A little bit more than a month has passed since Qlik Core went from beta to a live product. During this month we have
+focused on adding tools to support experienced and new developers in their journey to start working with Qlik Core.
+
+### corectl
+
+First versions of [corectl](https://github.com/qlik-oss/corectl) was released initial bug fixing, documentation and
+usability enhancements. The tool's aim is to act as a command line interface to the Qlik engine and can be used for
+querying and interacting with the engine instance, its apps and their data.
+
+![screenshot](./images/corectl.png)
+
+Main use case are:
+
+* View: List applications, view and query data within and evaluate expressions
+* Develop: Create, update and model a Qlik application
+* Script: Automate certain tasks like schedule a reload of an application
```

Comment:
I don't think this is true, corectl doesn't allow you to schedule anything

Reply:
True.. It should be *allow for automation* e.g. using cron.